modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -287,7 +287,9 @@ def location_phone(appointment_hash)
         end
 
         def time_zone(facility_id)
-          facility = Mobile::VA_FACILITIES_BY_ID["dfn-#{facility_id}"]
+          return nil unless facility_id
+
+          facility = Mobile::VA_FACILITIES_BY_ID["dfn-#{facility_id[0..2]}"]
```

Comment:
this is a temporary fix for the timezone issue discussed. these hardcoded facilities we use to determine timezone are only the vista ids. (3 numbers). but the location_id can be a child facility (442AB) which won't match anything. This is still problematic since a clinic to a parent facility can be in a different time zone but this will be solved in a separate issue. This fix will at least make it so the local start date is not the same as utc.